File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int current = 0;
        int sol = 0;

        for(int i=0;i<nums.length;i++) {
            if(nums[i]==0) {
                current = 0;
            } else {
                current +=1;
            }
            sol = Math.max(sol,current);
        }
        return sol;
    }
}

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        l = len(nums)
        count = 0
        ans = 0
        for i in range(l):
            if nums[i] == 1:
                count+=1
            else:
                count=0
            ans = max(ans,count)
        return ans
        
        